browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -9785,6 +9785,19 @@ index 350fdd497eaaab711059c54b5075015b19379368..bf5cd7221e1a131dd880d11e968b1327
      )
  endif ()
  
+diff --git a/Source/WebKit/Scripts/generate-unified-sources.sh b/Source/WebKit/Scripts/generate-unified-sources.sh
+index c67e2d1e3d5dd0b98a8775cea6ed9b78e593eb0a..7e5dd02f7e91e6264148d7024b3632750fb1bf50 100755
+--- a/Source/WebKit/Scripts/generate-unified-sources.sh
++++ b/Source/WebKit/Scripts/generate-unified-sources.sh
+@@ -14,7 +14,7 @@ if [ -z "${BUILD_SCRIPTS_DIR}" ]; then
+     fi
+ fi
+ 
+-UnifiedSourceCppFileCount=111
++UnifiedSourceCppFileCount=112
```

Comment:
Should UnifiedSource112.cpp be added to xcode project and UnifiedSources-output.xcfilelist similar to UnifiedSource111.cpp in https://trac.webkit.org/changeset/286077/webkit ?